backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/AttributeSetAggregator.java

```diff
@@ -37,6 +38,11 @@ void updateAggregatedValuesTo(@NonNull final IAttributeSet to)
 				.forEach(attributeAggregator -> attributeAggregator.updateAggregatedValueTo(to));
 	}
 
+	private static boolean isAggregable(@NonNull final I_M_Attribute attribute)
+	{
+		return !attribute.getValue().contains("Weight");
```

Comment:
pls use de.metas.handlingunits.attribute.weightable.Weightables.ATTR_ constants.
it's much more precise and easy to track

Reply:
even better, have a static ImmutableSet where u add all those AttributeCode(s) u want to ignore